requests/adapters.py

```diff
@@ -309,9 +309,7 @@ def get_connection(self, url, proxies=None):
             proxy_manager = self.proxy_manager_for(proxy)
             conn = proxy_manager.connection_from_url(url)
         else:
-            # Only scheme should be lower case
-            parsed = urlparse(url)
-            url = parsed.geturl()
+            # connection_from_url handles case conversions.
```

Comment:
As a result of this change the code is arguably more consistent with the `proxy` code path.